lint-staged.config.js

```diff
@@ -9,23 +9,18 @@ const tsConfigPath = './tsconfig.lint-staged.json'
 const tsConfig = (files) => `
 {
   "extends": "./tsconfig.json",
-  "include": [
```

Comment:
why do you removed this ? it includes all our custom typedefs, without them tsc may fail checking syntax on commit hook

Reply:
it wasnt working. not sure why we need to check the typedefs